### PR TITLE
issue82-added render icon to backend

### PR DESCRIPTION
**Pull request by qwertyroop**
There was no icon for the database tool render so I've added it

### CODE REVIEW

File: public/icons/skills/render.svg
Comment (Sygisk) [inline public/icons/skills/render.svg:1]: The logo looks like it's been modified.

Comment (danielcranney) [reply public/icons/skills/render.svg:1]: Unfortunately the icon shown did not meet the quality expectations, but has been replaced by a high quality SVG of the original logo.

**Review by danielcranney — COMMENTED** [review]
Changes absorbed into main branch.

**Comment by danielcranney** [comment]
PR closed as higher quality SVG added in place.

**Pull request closed without merging**